### PR TITLE
Meeting 2023-12-07 minutes

### PR DESCRIPTION
**Pull request by sthagen**
- [x] pre-meeting minutes draft
- [x] post-meeting revision (ready for review)

### CODE REVIEW

**Review by katrinaoneil — APPROVED** [review]

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+* No news
+
+## 3.3 Discuss the list of small non-breaking changes for SARIF v2.2
+
+ * Stefan: Explains his proposal for going forward technically per pull 
+           request [Seeding the SARIF v2.2 workplace - separating source and deliverables #599](https://github.com/oasis-tcs/sarif-spec/pull/599).
+ * All discuss
+ * David: Suggests that 
+   * at the January, 11 meeting the TC has the reference representation of v2.1 Errata 01 to approve it
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:113]: - Add comma after "meeting"
- Add "ready" before "to approve"

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+   * then, by the February, 8 meeting to have approved proposals to merge into the v2.2 working draft
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:114]: - Add comma after "meeting"
- Add "ready" before "to merge"
- Add period at end of sentence.

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+ * All agree to the plan 
+ * Michael: Hopes to have by end of June 2024 have the bulk of small changes for v2.2 merged into and then only have to polish for progressing the v2.2 towards OASIS Standard
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:116]: - Remove the second instance of "have"
- Remove "into"
- Add period at end of sentence

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+ * Michael: Issue [Suggestion: add an expiryDateUtc property to suppression instances #492](https://github.com/oasis-tcs/sarif-spec/issues/492)
+   * Suggests to continue with adding more examples and then have this proposal one of the first additions (v2.2)
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:118]: - Add "be" after "this proposal"
- Add period at the end of sentence

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+   * Points everyone to [OS SARIF v2.1.0 section 3.9 "Date/time properties"](https://docs.oasis-open.org/sarif/sarif/v2.1.0/errata01/os/sarif-v2.1.0-errata01-os-complete.html#_Toc141790703)
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:119]: Add period at the end.

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+ * Charles: Encourages to bring examples also for expanded ISO8601 forms
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:120]: Rewrite to: "Encourages author to add examples for expanded ISO8601 forms."

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+ * Stacy: Kindly requests to have someone go through this issue processing offline to get used to it
+ * Michael: Shows everyone some examples of how similar the html/pdf paragraphs look like the issue text rendered in GitHub
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:122]: - Add "when" after "like"
- Add "is" before "rendered"
- Add period at the end

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+* None
+
+# 5. Resolutions and Decisions reached (by 10 minutes prior to scheduled meeting end)
+
+## 5.1 End debate of other issues by 10 minutes prior to scheduled meeting end and follow the agenda from this point (Co-Chair David)
+
+## 5.2 Review of Decisions Reached (Co-Chair David)
+
+* DECISION to meet on February 8, 2024
+* DECISION to approve the issue [Suggestion: add an expiryDateUtc property to suppression instances #492](https://github.com/oasis-tcs/sarif-spec/issues/492) as discussed implementing the comment of David Malcolm and the implicit expansion of midnight commented by Charles
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:143]: Add period to end.

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+
+## 5.3 Review of Action Items (Co-Chair David)
+
+* Ongoing ACTION (from former meetings) on Charles to provide extensions to the roadmap document
+  [Future.md](https://github.com/oasis-tcs/sarif-spec/blob/main/Future.md) per pull request
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:148]: Add period to end.

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+* ACTION on Michael to triage on issue
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:149]: - Remove "on"
- Add period at end

File: meeting_minutes/231207_SARIF_TC_82.md [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+* ACTION on Michael to organize label renovation
```
Comment (stacywray) [inline meeting_minutes/231207_SARIF_TC_82.md:150]: Add period to end.

**Review by stacywray — CHANGES_REQUESTED** [review]
Minor edits and punctuation. Thanks!

**Comment by sthagen** [comment]
@stacywray I do not put full stops at the end of list items. Thanks for the corrections. Besides the action on Michael no semantics did change. The action was to triage old issues not all issues (that is what I remember).

**Review by stacywray — APPROVED** [review]

**Pull request merged**